### PR TITLE
added Ant neded for creation of jbpm-installers

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

Ant installation needed for creation of jbpm-installers during the community pipeline
<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: jenkins-slaves/ansible/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -150,6 +150,23 @@
   with_items:
     - "{{ maven_versions }}"
 
+### Ant ###
+-name: Download Ant archives
+  get url:
+    url: https://archive.apache.org/dist/ant/binaries/apache-ant-{{ant_version}}-bin.tar.gz
```
Comment (winklerm) [inline jenkins-slaves/ansible/roles/common/tasks/main.yml:156]: I think the spaces inside {{ and }} should be preserved:
```suggestion
    url: https://archive.apache.org/dist/ant/binaries/apache-ant-{{ ant_version }}-bin.tar.gz
```

File: jenkins-slaves/ansible/roles/common/tasks/main.yml [hunk 2]
```diff
@@ -150,6 +150,23 @@
...
+    dest: /tmp/downloads
+    owner: root
+    group: root
+
+-name: Extract Ant archives
+  unarchive:
+    copy: no
+    src: /tmp/downloads/apache-ant-{{ant_version}}-bin.tar.gz
```
Comment (winklerm) [inline jenkins-slaves/ansible/roles/common/tasks/main.yml:164]: The same here:
```suggestion
    src: /tmp/downloads/apache-ant-{{ ant_version }}-bin.tar.gz
```

**Review by winklerm — CHANGES_REQUESTED** [review]
Looks ok, except the spaces inside {{ and }} for variables - that I think is a convention (not sure if a requirement) by Ansible.

**Comment by mareknovotny** [comment]
just curious why or what do we need ant for?

**Comment by mbiarnes** [comment]
@mareknovotny Ant is needed for the creation of the jbpm-installers when a community release was done

**Review by winklerm — APPROVED** [review]
LGTM, thanks!

**Pull request merged**